src/Components/TopBanner/TopBanner.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  Button,
+  Card,
+  CardBody,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import '../../Routes/SandboxPage/SandboxPage.scss';
+
+const TopBanner = () => {
+  return (
+    <>
+      <Card
```

Comment:
Same as above, extra return statement and empty element
```suggestion
const TopBanner = () => (
  <Card
```